### PR TITLE
Nithya peachtree city

### LINKED ISSUE #13: Nithya My List
- [ ] Unchecked Testing
- [x] Checked
- [x] Working

### PR DESCRIPTION
**Pull request by nithyasomanath**
:beers:
:car:
Testing

Closes :#2
Resolves #13

### CODE REVIEW

File: myfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Testing new file contents
```
Comment (robinmathewkurian) [inline myfile]: Increase font size?

File: myfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+Adding one more line
+
```
Comment (udenjoe) [inline myfile:4]: Missing item. Should do research on places to eat.

File: myfile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Testing new file contents
+Adding one more line
```
Comment (robinmathewkurian) [inline myfile]: ##Make it a subtitle

**Review by robinmathewkurian — COMMENTED** [review]
Comments updated

File: myfile (hunk 2 above)
Comment (nithyasomanath) [reply myfile:4]: sure :+1:

**Review by udenjoe — CHANGES_REQUESTED** [review]
Need to update myfile with missing info.

File: myfile [hunk 4]
```diff
@@ -0,0 +1,5 @@
+New Areas are 
+Mc Donald
+Panera Bread
+
```
Comment (udenjoe) [inline myfile:4]: Did I leave a comment here?

Comment (nithyasomanath) [reply myfile:4]: No you did not :beer:

**Comment by nithyasomanath** [comment]
:dark_sunglasses:

**Pull request merged**